modules/nf-core/gatk4/genotypegvcfs/main.nf

```diff
@@ -8,12 +8,12 @@ process GATK4_GENOTYPEGVCFS {
         'biocontainers/gatk4:4.5.0.0--py36hdfd78af_0' }"
 
     input:
-    tuple val(meta), path(gvcf), path(gvcf_index), path(intervals), path(intervals_index)
-    path  fasta
-    path  fai
-    path  dict
-    path  dbsnp
-    path  dbsnp_tbi
+    tuple val(meta), path(input), path(gvcf_index), path(intervals), path(intervals_index)
```

Comment:
Don't forget to update the meta.
Also should gvcf_index be renamed too?

Reply:
on it! and no because the index is only used in the gvcf case :p